src/js/actions/libraries.js

```diff
@@ -1267,6 +1267,7 @@ define(function (require, exports) {
         descriptor.addListener("toolModalStateChanged", _toolModalStateChangedHandler);
 
         return apiLoadPromise
+            .timeout(3000, "CC Libraries API load timeout, please don't restart and notify the chatroom!")
```

Comment:
please don't (restart and notify), or please don't restart, but DO notify.  I'm confused.  JUUUST kidding.